DependencyInjection/FOSCommentExtension.php

```diff
@@ -54,6 +56,29 @@ public function load(array $configs, ContainerBuilder $container)
             $this->loadAcl($container, $config);
         }
 
+        // To be removed when dependency on Symfony DependencyInjection is bumped to 2.6
+        if (!interface_exists('Symfony\Component\Security\Core\Authentication\Token\Storage\TokenStorageInterface')) {
+            $container->setDefinition('security.token_storage', new Reference('security.context'));
+        }
+
+        // To be removed when dependency on Symfony DependencyInjection is bumped to 2.6
+        if (!interface_exists('Symfony\Component\Security\Core\Authorization\AuthorizationCheckerInterface')) {
```

Comment:
technically, it is totally valid to use ``symfony/security-bundle`` 2.5 with ``symfony/security-core`` 2.7. and what you really need to detect is whether SecurityBundle is defining the new services (i.e. SecurityBundle being at 2.6+). So the right way to detect it is to check whether the service ``security.authorization_checker`` is defined (which needs to be done in a compiler pass, not in a DI extension, as the DI extension does not receive the main container builder). See https://github.com/liip/LiipTranslationBundle/pull/39/files#diff-c2c20f7d6267a9f0628c1dda9a00437f for such an example